src/__tests__/TimelineComponent.test.tsx

```diff
@@ -635,12 +635,6 @@ describe('<TimelineComponent /> Unit Tests', () => {
     });
   });
 
-  describe('updateBars()', () => {
-    it('dummy test', () => {
-      expect(updateBarsSpy).toHaveBeenCalled();
-    });
-  });
-
   describe('moveChart()', () => {
     it('dummy test', async () => {
       wrapper.instance().drawTimeline();
```

Comment:
Can we delete this one? Has a lot of commented out code and is still called dummy test

Reply:
I was wondering about that too. I just asked Mesa and he said it was fine to get rid of it, so it's been removed.